src/contracts/core/AllocationManager.sol

```diff
@@ -164,6 +170,11 @@ contract AllocationManager is
                     // Add the operatorSet to the deallocation queue
                     deallocationQueue[msg.sender][allocation.strategy].pushBack(operatorSetKey);
                 } else if (info.pendingDiff > 0) {
+                    require(
+                        avsDirectory.isOperatorSetStrategy(allocation.operatorSets[j], allocation.strategy),
```

Comment:
The net effect here is that you should ONLY be able to allocate to strategies that are whitelisted but can deallocate regardless. I think this is great. Can we add this assumption to the interface?

Reply:
You mean document it?

Reply:
Ya in the interface comments